client/webpack.config.js

```diff
@@ -22,24 +20,29 @@ const config = {
   },
 
   resolve: {
-    extensions: ['', '.js', '.jsx'],
-    alias: {
-      react: path.resolve('./node_modules/react'),
-      'react-dom': path.resolve('./node_modules/react-dom'),
-    },
+    // https://webpack.js.org/guides/migrating/#resolve-extensions
+    extensions: ['.js', '.jsx'],
+
+    // Remove alias as this was needed when we had nested node_modules sub-dependencies
+    // alias: {
+    //   react: path.resolve('./node_modules/react'),
+    //   'react-dom': path.resolve('./node_modules/react-dom'),
+    // },
   },
   plugins: [
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify(nodeEnv),
-      },
-    }),
+    // https://webpack.js.org/plugins/environment-plugin
+    new webpack.EnvironmentPlugin({ NODE_ENV: 'development' }),
```

Comment:
NOTE: It didn't work for me on fng when I passed object as argument. Maybe it's bug or something specific to fng, better recheck.